apis/deployer/helm/types_provider.go

```diff
@@ -53,6 +62,17 @@ const (
 	UpdateStrategyPatch  UpdateStrategy = "patch"
 )
 
+// HealthChecksConfiguration contains the condiguration for health checks.
+type HealthChecksConfiguration struct {
+	// Disable allows to disable health checks.
+	// +optional
+	Disable bool `json:"disable,omitempty"`
+	// TimeOutSeconds is the time to wait in seconds before giving up on a resource to be healthy.
+	// Defaults to 60s.
+	// +optional
+	TimeOutSeconds int32 `json:"timeOutSeconds,omitempty"`
```

Comment:
how about making that field a string and then parse it with `time.ParseDuration`.
With that it would be possible to define human readable values like `5m`

Reply:
I've changed this and also added validation.